web-client/package.json

```diff
@@ -43,7 +43,10 @@
   },
   "scripts": {
     "start": "SKIP_PREFLIGHT_CHECK=true react-app-rewired start",
-    "build": "NODE_ENV=development react-app-rewired build",
+    "build": "react-app-rewired build",
```

Comment:
This is mostly the section I need help with in terms of using env vars in react.

`web-client/src/firebase/index.ts` contains the logic that should leverage the env var to load the proper config.